draft-ietf-quic-http.md

```diff
@@ -237,6 +237,11 @@ response may contain zero or more header blocks containing the message headers
 of informational (1xx) HTTP responses (see {{!RFC7230}}, Section 3.2 and
 {{!RFC7231}}, Section 6.2).
 
+At any point during a response message, PUSH_PROMISE frames MAY also be present
```

Comment:
nit: It might be clearer to use the word interleaved in there somewhere. Suggested rewording: "A response message MAY be interleaved with PUSH_PROMISE frames that indicate a pushed resource related to the response." Otherwise, LGTM.